VelocidiSDK/VelocidiSDK/VSDKConfig.m

```diff
@@ -4,25 +4,23 @@
 
 @implementation VSDKConfig
 
-- (instancetype)initWithHosts: (NSString *)trackingHost :(NSString *)matchHost {
+- (instancetype)initWithTrackingHost: (NSString *)trackingHost
+                                    : (NSString *)matchHost {
     if(self = [super init]) {
-        self.trackingHost = [[NSURL alloc] initWithString:trackingHost];
-        self.matchHost = [[NSURL alloc] initWithString:matchHost];
-        
+        self.trackingUrl = [[NSURLComponents alloc] initWithString:trackingHost];
+        self.matchUrl = [[NSURLComponents alloc] initWithString:matchHost];
+        self.trackingUrl.path = @"/events";
```

Comment:
The path should be empty, so if we either append it or replace it should be the same. Can you expand on this?

Reply:
I tried using [Mocky as described in the README.md](https://github.com/velocidi/velocidi-ios-objc-sdk/tree/master/Examples#capture-http-events) and it didn't work, because Mocky produces an URL like this: https://www.mocky.io/v2/5185415ba171ea3a00704eed

Reply:
I see, but you provided the full URL (not the `host`), which caused the path portion of the URL to be dropped. I'll rename the method to `initWithTrackingBaseUrl` to be more comprehensive and append the `/events` at the end.

Reply:
Please check a614b6f